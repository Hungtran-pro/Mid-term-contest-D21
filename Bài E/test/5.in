10
680 8
fh?dcahdcahbacf?begeffcf?hbeaadbea?cchc?beeg?adec??ehhhdg?gd?bh?chdfbdhdffggc?ceccdebcffgc?fbgffhaddh?af?hbbfhcf???edgadgbegcb?cg?hgdb?dadceedeahagb?fd?hfacfb?gfdahdeache?aadf?ga?dbcffabeddgbgbdfhhedgahbgh?aaggh??bhgahbahcahheeebcbhecaebgg?ghedgdgb?cagga?efhccedbhcgegagcfccgfddbcdcebaacbbbgadgfgbdaeffhdec?bbbcbhf?bccebcd?debg?agdbgh?cddhgggdedddeab?ddhfdedacdhaaffd?adgcaee?g?gecgghc?fa?fahbaceabhdabfaadfhfecdhghbfdgafcc?gcabgab?dcddehfgabdcheh?g?cfcfeab?abdd?adcbcffd?cchdhhdccagefada?eacc?fbfb?cffbdge?dfcdh??fg?acce?dhehgdegehd?gfa?bgdefgdfdc?gghef?fce?fadadcde?dehc?gaga?fhhhd?cd?gd?hhg?dghhgefedgaggcbbcea??cbd?bdhe?befccf?gegbf?cbfg?facdbbed?c?deagab?hd?acbb?bfeadegf?ebb
631 9
?hgae?hebhbgdeeedechcggbgddhhbehfhc?c?gii?ghihicbcfabhed?ebifhc?efbchbi?bfbaihbabdachhdcbfibcfdabhcghccbbe?cihfh?ihgihgii?hcfheid?ibfief?e??gd?cehhb?ba?fdc?h?fhggfgcb?d?edaegfaaccbfghfafcc?dhhiehcf?gcebdchahdhegbabefcgfeggifc?g?fb??ffgfag?hbbhhdhchgdi?iechaddachdifgffii?haighcbb?hccehhibgghhbehgbifbaafid?f?bfe??d?gacdbbc?acfie?defhciecgagg?hcheaeecie?acciiifbiib?cbeaehi?dfgh?ebhcaehfeg?adhhgh?hegi??i?d?ciaaaffbbeaded?giaah?bdhda?eahifa?eidigbig?ddgdfhdidcffc?idbcgg?ih?bcbgfib?fhgbbhdicfbffagbfaa??gbgfbciei?d?gchbeahageca?ahhcdchedaiaaahc?gdcgeaca??bfbcdg?ahcgibdbagibaicchch?ciigfai?fa?hccdgaebhcdifb?gcddcibffibabgfifadifa?e
554 6
?ad??aafeaddc?bba?f?abcfcedbacacd?bbbb?cdcbabeacdfaa??dffdafdefafaeceedddcdca??dadffefbefcebcc?d?feb?fedddaaaaf?cebaffad?bdabecc?fcfceeafdeb?bbfc?abeb?aecbebfaddabfaca??b?eaed?dddbcce?fffcbfb?f?eaacbabeeaddeeabcccfffa?f?fececeabefcecfea?bddbbdfadd?e?bfdfaffdcaffeebedacdfeffbaaeaaa?dbec?f?bde?ada?dfceefefabeadcaebddaa??cecd?e?adadca?ffcbdd?f?fb??afb?eadfdeecceeaabdfddbec?d?bacccfd?b?eaefdedb?fe??eabccbaeafaeccfa??d?b?edb??beb?a?efbcdebae?de?decca?c??aeecaa?e?cbfbccbdccc?dbef?df?aabedadc?d?f?dcbbebcef?bbfa?eaaac?f?bb?fdbbcfb?fdacafbcddc?d???ddfbcef?f
722 10
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiieeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiieeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeehhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb??
653 6
bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb????????????????????????????????????????????????????????????????????????????????????????????????????????????eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee????????????????????????????????????????????????????????????????????????????????????????????????????????????ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff?????
639 6
??????????????????????????????????????????????????????????????????????????????????????????????????????????eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee??????????????????????????????????????????????????????????????????????????????????????????????????????????bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee???
958 6
fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa???????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff????
752 10
jjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccceeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeedddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddgggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiijjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb??
693 10
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiieeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee?????????????????????????????????????????????????????????????????????aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaagggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff???
801 6
?????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
