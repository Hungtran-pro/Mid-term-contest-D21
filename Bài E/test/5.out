3
3
4
0
54
53
53
1
34
133
